tsdb/isolation.go

```diff
@@ -104,6 +117,10 @@ func (i *isolation) lowWatermarkLocked() uint64 {
 // State returns an object used to control isolation
 // between a query and appends. Must be closed when complete.
 func (i *isolation) State(mint, maxt int64) *isolationState {
+	if i.disabled {
+		return &isolationState{}
+	}
```

Comment:
We _need_ to track open reads :) disabling isolation would not disable tracking of reads